inc/functions/files.php

```diff
@@ -1273,24 +1273,23 @@ function rocket_get_filesystem_perms( $type ) {
 }
 
 /**
- * Try to find the correct wp-config.php file, support one level up in filetree
+ * Try to find the correct wp-config.php file, support one level up in file tree.
  *
  * @since 2.1
  *
- * @return string|bool The path of wp-config.php file or false if not found
+ * @return string|bool The path of wp-config.php file or false if not found.
  */
 function rocket_find_wpconfig_path() {
 	/**
-	 * Filter the wp-config's filename
+	 * Filter the wp-config's filename.
 	 *
 	 * @since 2.11
-	 * @author Maxime Culea
 	 *
 	 * @param string $filename The WP Config filename, without the extension.
 	 */
 	$config_file_name = apply_filters( 'rocket_wp_config_name', 'wp-config' );
-	$config_file      = ABSPATH . $config_file_name . '.php';
-	$config_file_alt  = dirname( ABSPATH ) . '/' . $config_file_name . '.php';
+	$config_file      = rocket_get_constant( 'ABSPATH' ) . $config_file_name . '.php';
+	$config_file_alt  = dirname( rocket_get_constant( 'ABSPATH' ) ) . '/' . $config_file_name . '.php';
```

Comment:
2 Optimizations:

1) Move this line down after first `if` and change the `elseif` to an `if`.

Why? 

Notice that if the first conditional is true, then it returns `$config_file`. That means we don’t need to do the alt version of it. Saves memory and processing time.

2) Concatenation via embedded variables.

It’s faster to concatenation multiple string segments by embedding the variable:

```php
“/{$config_file_name}.php”;
```